worlds/tracker/TrackerClient.py

```diff
@@ -69,7 +69,6 @@ def _cmd_reset_manually_collect(self):
 
 
 class TrackerGameContext(CommonContext):
-    from kvui import GameManager
     game = ""
```

Comment:
```suggestion
    import kvui
    game = ""
```

Reply:
why? the PR works, lol

Reply:
to make sure this isn't a misunderstanding,
https://github.com/FarisTheAncient/Archipelago/blob/tracker/worlds/tracker/TrackerClient.py#L187
it is imported another time for when its needed for more than type hinting

Reply:
![image](https://github.com/FarisTheAncient/Archipelago/assets/24858499/7488b24a-0438-4f0a-9f8c-8da9bb7dd8bd)
Well, the github tests don't agree with you, but I wouldn't be surprised if there was just some python fuckery going on lol

Reply:
looks like i found it, it was type checking shenanigans